content/teknologi/altinnstudio/altinn-api/app-api/process/_index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Process
+description: The process API endpoints to lead an instance through the defined process.
+toc: true
+tags: [api]
+weight: 180
+---
+
+## Overview
+
+Application has a process definition that specifies start events, end events, tasks and the allowed flows (transitions) between these.
+A process is started by the application, which selects a start event to start and follows the sequence flow to the first task and
+creates a current task object to holde the process state.
+
+![Flowchart for process](mvp-process.png "Process")
```

Comment:
Is the image missing here?

Reply:
I think it goes for all the images on this page. They aren't loading on my image at least.